lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -859,7 +865,36 @@ private function Annotation()
             );
         }
 
-        $values = $this->MethodCall();
+        $arguments = $this->MethodCall();
+
+        $positionalArguments = $arguments['positional_arguments'] ?? [];
+        $namedArguments      = $arguments['named_arguments'] ?? [];
+
+        if (
```

Comment:
Could you extract this block into a private helper please?


    $values = $this->assembleValues($arguments, $name);

Method name could be better

Reply:
Updated!